src/store/reducers/index.js

```diff
@@ -72,7 +73,7 @@ export const reducer = (
 			// Otherwise merge attributes into state
 			block.attributes = nextAttributes;
 
-			return { blocks: blocks, refresh: ! state.refresh };
+			return { blocks: blocks, initialHtmlHash: state.initialHtmlHash, refresh: ! state.refresh };
```

Comment:
Nitpicky, but for all these updates we can use the spread operator and property shorthand:

```suggestion
			return { ...state, blocks, refresh: ! state.refresh };
```

Reply:
I think it's easier to read and understand when you only make explicit what is changing

Reply:
Done with 5180b0c.